packages/react/src/themes/teams-high-contrast/components/Menu/menuVariables.ts

```diff
@@ -1,13 +1,26 @@
+import { pxToRem } from '../../../../lib'
+
 export interface MenuVariables {
   color: string
   activeColor: string
   activeBackgroundColor: string
+  selectedBackgroundColor: string
+
+  verticalMenuBackgroundColor: string
+  menuDividerHeight: string
+  menuBorderWidth: string
 }
 
 export default (siteVars: any): MenuVariables => {
   return {
     color: siteVars.white,
     activeColor: siteVars.black,
     activeBackgroundColor: siteVars.accessibleYellow,
+    selectedBackgroundColor: siteVars.accessibleCyan,
+
+    verticalMenuBackgroundColor: siteVars.colors.black,
+
+    menuDividerHeight: pxToRem(2),
+    menuBorderWidth: pxToRem(2),
```

Comment:
NIT: ~~`menuBorderWidth`~~ -> `borderWidth` since we're already in the context of the menu